components/eventsrv-protocol/protocols/event.proto

```diff
@@ -65,12 +65,16 @@ message PackageIdent {
 message CensusEntry {
   /// Member-ID of originating Supervisor
   optional string member_id = 1;
+
   /// Service name
   optional string service = 2;
   /// Service group name
   optional string group = 3;
   /// Service group organization name
   optional string org = 4;
+  // NOTE: service group application and environment are further down
```

Comment:
Protobufs have you assign numbers for a field so you can order them however you want inside the file. Feel free to just pull those up here and remove this comment :)